src/styles/index.scss

```diff
@@ -1,3 +1,253 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+}
```

Comment:
The `body` selector already sets the `font-family` to 'Roboto, Arial, sans-serif'. Consider removing the `.page` class's `font-family` declaration to avoid redundancy unless it's specifically needed for a different context.